cartography/intel/aws/inspector.py

```diff
@@ -20,18 +22,18 @@
 @timeit
 @aws_handle_regions
 def get_inspector_findings(
-    session: boto3.session.Session,
-    region: str,
-    current_aws_account_id: str,
-) -> List[Dict]:
+        session: boto3.session.Session,
+        region: str,
+        current_aws_account_id: str,
+) -> List[Dict[str, Any]]:
     '''
     We must list_findings by filtering the request, otherwise the request could tiemout.
     First, we filter by account_id. And since there may be millions of CLOSED findings that may never go away,
     we will only fetch those in ACTIVE or SUPPRESSED statuses.
     list_members will get us all the accounts that
     have delegated access to the account specified by current_aws_account_id.
     '''
-    client = session.client('inspector2', region_name=region)
+    client = session.client('inspector', region_name=region)
```

Comment:
I think you meant to keep v2. list_findings in v1 doesn't return all the data we need.
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/inspector/client/list_findings.html